include/BlazeIterative.hpp

```diff
@@ -12,4 +12,6 @@
 
 #include "solvers/solvers.hpp"
 
+#define epsilon 1.0e-8
```

Comment:
Please use only capital letters to define constants, like EPSILON

Reply:
Changed them.